spec/specification/json_spec.rb

```diff
@@ -16,6 +16,7 @@ module Pod
             'tvos' => nil,
             'watchos' => nil,
           },
+          'test_specification' => false,
```

Comment:
Could `false` be implicit for a null value? All older JSON pod specs would have that already, and so only the new specs with tests would end up with `'test_specification' => true,`. 

I'm pretty sure CocoaPods ignores attributes it doesn't understand (which is why I could add the Swift version) so backwards compact for adding this should be 👍